src/features/identityCheck/pages/profile/StatusFlatList.tsx

```diff
@@ -0,0 +1,162 @@
+import { AnimatedViewRefType, createAnimatableComponent } from 'libs/react-native-animatable'
+import { theme } from 'theme'
+import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
+import { Li } from 'ui/components/Li'
+import { RadioSelector } from 'ui/components/radioSelector/RadioSelector'
+import { Spinner } from 'ui/components/Spinner'
+import { getSpacing, Spacer, Typo } from 'ui/theme'
+
+const GRADIENT_HEIGHT = getSpacing(30)
+const VIEWABILITY_CONFIG = { itemVisiblePercentThreshold: 100 }
+
+export type StatusForm = {
+  selectedStatus: ActivityIdEnum | null
+}
+
+interface Props {
+  handleSubmit: UseFormHandleSubmit<StatusForm>
+  isLoading: boolean
+  selectedStatus: ActivityIdEnum | null
+  submitStatus: (formValues: StatusForm) => Promise<void>
+  titleID: string
+  control: Control<StatusForm>
+  headerHeight: number
+}
+
+export function StatusFlatList({
+  handleSubmit,
+  isLoading,
+  selectedStatus,
+  submitStatus,
+  titleID,
+  control,
+  headerHeight,
+}: Props) {
+  const { activities } = useActivityTypes()
+
+  const gradientRef = useRef<AnimatedViewRefType>(null)
+
+  const { onViewableItemsChanged } = useOnViewableItemsChanged(gradientRef, activities ?? [])
+
+  const [bottomViewHeight, setBottomViewHeight] = useState(0)
+
+  function onBottomViewLayout(event: LayoutChangeEvent) {
+    const { height } = event.nativeEvent.layout
+    setBottomViewHeight(height)
+  }
+
+  const renderItem: ListRenderItem<ActivityResponseModel> = ({ item }) => (
+    <View accessibilityRole={AccessibilityRole.RADIOGROUP} accessibilityLabelledBy={titleID}>
+      <Li key={item.label}>
+        <Controller
+          control={control}
+          name="selectedStatus"
+          render={({ field: { value, onChange } }) => (
+            <RadioSelector
+              checked={item.id === value}
+              label={item.label}
+              description={item.description}
+              onPress={() => onChange(item.id)}
+            />
+          )}
+        />
+        <Spacer.Column numberOfSpaces={3} />
+      </Li>
+    </View>
+  )
+
+  return (
+    <React.Fragment>
+      {activities ? (
+        <FlatListContainer>
+          <FlatList
+            scrollIndicatorInsets={{ right: 1 }} // Corrects scrollbar in the middle
+            contentContainerStyle={flatListStyles}
+            onViewableItemsChanged={onViewableItemsChanged}
+            viewabilityConfig={VIEWABILITY_CONFIG}
+            data={activities}
+            keyExtractor={(item) => item.label}
+            ListHeaderComponent={
+              <React.Fragment>
+                <View style={{ height: headerHeight }} />
+                <Spacer.Column numberOfSpaces={2} />
+                <Typo.Title3>{'Sélectionne ton statut'}</Typo.Title3>
+                <Spacer.Column numberOfSpaces={8} />
+              </React.Fragment>
+            }
+            renderItem={renderItem}
+          />
```

Comment:
Mettre ça et voir si ça affiche bien avec l'inspecteur

```suggestion
          <FlatList
            scrollIndicatorInsets={{ right: 1 }} // Corrects scrollbar in the middle
            contentContainerStyle={flatListStyles}
            onViewableItemsChanged={onViewableItemsChanged}
            viewabilityConfig={VIEWABILITY_CONFIG}
            data={activities}
            keyExtractor={(item) => item.label}
            ListHeaderComponent={
              <React.Fragment>
                <View style={{ height: headerHeight }} />
                <Spacer.Column numberOfSpaces={2} />
                <Typo.Title3>{'Sélectionne ton statut'}</Typo.Title3>
                <Spacer.Column numberOfSpaces={8} />
              </React.Fragment>
            }
            renderItem={renderItem}
          />
```

Reply:
Je pense que c'est bon:
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/da2496ac-79bf-42c7-a453-34093532f29c">

Reply:
On devrait avoir UL > LI > DIV et pas UL > DIV > LI
Enfin d'un point de vu HTML